### PR TITLE
add font size ramp migration guide and update utilities page

### PR DESCRIPTION
**Pull request by josedialpad**
Update of the font size ramp (3/3).
Divided the task into 3 parts to separate the changes:

1. Update font-size ramp.
2. Migrate usages in Dialtone.
3. Update Utilities page and migration guide.

This PR covers part 3.

## Pull Request Checklist

 - [x] Ask the contributors if a similar effort is already in process or has been solved.
 - [x] Review the [contribution guidelines](https://github.com/dialpad/dialtone/blob/staging/.github/CONTRIBUTING.md).
 - [ ] Use `staging` as your pull request's base branch. (All PRs using `production` as its base branch will be declined).
 - [ ] Ensure all `gulp` scripts successfully compile.
 - [ ] Update, remove, or extend all affected documentation.
 - [ ] Ensure no private Dialpad links or info are in the code or pull request description (Dialtone is a public repo!).

### CODE REVIEW

**Comment by braddialpad** [comment]
Hmm why am I seeing all the changes from the other PR in here?

**Comment by josedialpad** [comment]
> Hmm why am I seeing all the changes from the other PR in here?

Sorry, I've merged the part 2 in the base PR and was updating this. Now it should be ready for reviewing.

File: migration_guide/MIGRATION_GUIDE.md [hunk 1]
```diff
@@ -212,3 +212,44 @@ Check updates to confirm desired rendering and alignment to contrast accessibili
 ### 6. Font updates
 
 Remove any usages of deprecated `.d-fw-thin`, `.d-fw-light` and `.d-fw-dark` classes.
```
Comment (francisrupert) [inline migration_guide/MIGRATION_GUIDE.md:214]: This was missed in previous Font Stack PR from @juliodialpad, but the "`d-fw-dark`" reference was meant to be `d-fw-black` (I'm pretty sure `fw-dark` didn't exist, right?).

Which now makes me realize...

We should also make sure we're replacing any instances of...

* `fw-thin` and `d-fw-light` to `fw-normal`
* `fw-black` to `fw-bold`

...and update this "Font updates" portion of Migration Guide to reflect that.

Comment (josedialpad) [reply migration_guide/MIGRATION_GUIDE.md:214]: @francisrupert good catch, created an issue to work on this updates in a different PR https://dialpad.atlassian.net/browse/DT-669

Comment (francisrupert) [reply migration_guide/MIGRATION_GUIDE.md:214]: Coolness.

**Review by francisrupert — APPROVED** [review]

**Pull request merged**